numba/unicode.py

```diff
@@ -343,6 +347,25 @@ def _find(substr, s):
     return -1
 
 
+_WHITESPACE_SPACE = ord(' ')
+_WHITESPACE_NEWLINE = ord('\n')
+_WHITESPACE_RETURN = ord('\r')
+_WHITESPACE_TAB = ord('\t')
+_WHITESPACE_FORMFEED = ord('\f')
+_WHITESPACE_VERTICAL_TAB = ord('\v')
+
+
+@njit
+def _is_whitespace(code_point):
+    # unrolling this for speed
+    return code_point == _WHITESPACE_SPACE or \
```

Comment:
These probably need including: https://github.com/python/cpython/blob/5105483acb3aca318304bed056dcfd7e188fe4b5/Objects/unicodetype_db.h#L5996-L6031

Reply:
ok, should be fixed